src/components/Account/Account.tsx

```diff
@@ -282,7 +285,11 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
             formSection({
               label: t('account.other_registration_details'),
               saveButton: t('account.update_consents'),
-              onSubmit: (values) => updateConsents(formatConsentsFromValues(publisherConsents, values.consentsValues)),
+              onSubmit: (values) => {
```

Comment:
Seems like the whole onSubmit logic is duplicated twice too, with just the `terms` boolean different. Could easily be abstracted into 1 function.